### DISCUSSION TITLE
How do you get a file to load from a relative path?

### DISCUSSION THREAD

**Discussion by vinny-888** [Ironclad/rivet#112] [Q&A]
I get file not found error.

The ./prompt is at the same level as the Rivet project file

**Comment by abrenneke** [comment]
I think this is going to take a bit of work to support, assuming it makes sense to support it. Let me think about the best way to make this work for both the app and the libraries 🤔

**Comment by vinny-888** [comment]
Use case is to have a rivet project file in the root of a repo and have a prompt folder where I load the system prompts from.

This would make it easier to edit prompts as they would each have their own file and commit history

**Reply by abrenneke** [reply]
Yeah that sounds similar to what #78 is talking about
